content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md

```diff
@@ -24,7 +24,11 @@ The attribute used as identifier will be displayed in read-only in the Supplier
 :::
 
 ::: success
-Only the attributes marked as 'required' are taken into account for the completeness of products. The nice to have attributes are not mandatory to synchronize the product. 
+Only the attributes marked as 'required' are taken into account for the completeness of products. The nice to have attributes are not mandatory to synchronize the product.
+:::
+
+::: warning
+Custom attributes are not supported by the Onboarder. Adding them in the families linked with your supplier could cause issues. 
```

Comment:
Does it mean that we CAN import custom attributes, but that it WILL cause issues? If that's so, is it possible to describe the potential issues that might arise?

Reply:
For now, it is possible to add custom attributes to families that are linked to suppliers. Normally, we shouldn't accept the custom attributes in the Onboarder to avoid the issues. But we would like to warn clients they shouldn't use custom attributes for their suppliers.